src/main/java/ru/itis/controllers/WishListController.java

```diff
@@ -35,8 +35,8 @@ public WishList getWishList(@PathVariable("list-id") Long listId, @RequestHeader
     @PostMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Add new item to wish list")
-    public void addNewItem(@PathVariable("list-id") Long listId, ItemForm itemForm, @RequestHeader(name = "AUTH") String token) {
-        itemService.addNewItem(itemForm, listId);
+    public void addNewItem(@PathVariable("list-id") Long listId, ItemDto itemDto, @RequestHeader(name = "AUTH") String token) {
```

Comment:
You have "AUTH" as a variable and as a property, why hardcoding it?

Reply:
If I remove name of the header, it won't be recognized -> 403 error